frontend/src/js/controllers/authCtrl.js

```diff
@@ -52,6 +56,28 @@
             vm.loginContainer.removeClass('low-screen');
         };
 
+        // toggle password visibility
+        vm.togglePasswordVisibility = function() {
```

Comment:
In all the toggle functions just do this for toggling their respective variables:
$scope.canShowPassword = !$scope.canShowPassword;

Please clean up all the newly added js code(toggle functions and their associated variables).

Reply:
@galipremsagar done